### ISSUE TITLE
Cancel Trip workflow

### ISSUE THREAD

**Issue by nitishMehrotra** [MOOVE-Network/moove-webapp#43]
**Description:**

Should cancelling a trip follow the same path as changing a trip?

**Comment by laurablamo** [comment]
From a UX perspective they are the same thing

**Comment by mustaqsingh** [comment]
Yes, for now lets have the approval mechanism there for cancellations.

**Comment by jasminelinblamo** [comment]
Following is the updated cancel trip flow. I'll update InVision and Zeplin after @nitishMehrotra and @yurixus review.

![cancel trip flow](https://cloud.githubusercontent.com/assets/25039197/21908375/5bac5852-d8c8-11e6-88bd-8ef8b62afd95.png)

**Comment by nitishMehrotra** [comment]
@jasminelinblamo as @laurablamo and @mustaqsingh suggested the workflow for cancellation will be same as workflow for requesting a change in time of the trip.
